src/components/EventCalendar/EventCalendar.module.css

```diff
@@ -19,6 +19,32 @@
   font-weight: bold;
   border: 0px;
 }
+
```

Comment:
Instead of styling the scrollbars individually in everyfile, create a common CSS file in `src/assets/css` folder

Reply:
> Instead of styling the scrollbars individually in everyfile, create a common CSS file in `src/assets/css` folder and load it inside `src/index.ts`

Sir, can you tell me how to use the styles from app.css? I am facing some issues while using it.

Reply:
Sir, can you tell me how to use the styles from app.css? I am facing some issues while using it.